emulsify.drush.inc

```diff
@@ -0,0 +1,299 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush hooks. Inspired by Aeon drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function emulsify_drush_command() {
+  $items = array();
+
+  $items['emulsify'] = array(
+    'description' => 'Create an Emulsify theme.',
+    'arguments' => array(
+      'name'         => 'The name of your theme.',
```

Comment:
We might want to specify "human-readable name" here. Not a huge deal, but might help usability.